TestON/drivers/common/cli/quaggaclidriver.py

```diff
@@ -23,7 +23,20 @@ def __init__( self ):
     def connect( self, **connectargs ):
         for key in connectargs:
             vars( self )[ key ] = connectargs[ key ]
+        self.name = self.options[ 'name' ]
```

Comment:
@pingping-lin can you take a look at these driver changes since you also use quagga?